src/components/status__cookie.pug

```diff
@@ -0,0 +1,3 @@
+.status--banner.theme--caution.cookie#cookie
+  .container
+    p Parliament.uk use cookies to make the site simpler. #[+link('#', 'Find out more about cookies.')]
```

Comment:
Last full stop should be outside the link.

Reply:
updated